### PR TITLE
Keychron k6 add device conditions

### PR DESCRIPTION
**Pull request by sourabhv**

### CODE REVIEW

File: public/json/keychron-k6-shortcuts.json [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
         {
+          "conditions": [
+            {
+              "type": "device_if",
+              "identifiers": [
+                {
+                  "product_id": 591,
+                  "vendor_id": 1452,
+                  "is_pointing_device": false,
+                  "is_keyboard": true
```
Comment (MuhammedZakir) [inline public/json/keychron-k6-shortcuts.json:17]: Either one is enough.

File: public/json/keychron-k6-shortcuts.json [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
+            }
+          ],
           "type": "basic",
           "from": {
             "key_code": "escape",
             "modifiers": {
-              "mandatory": [
-                "shift"
-              ]
+              "mandatory": ["shift"]
```
Comment (MuhammedZakir) [inline public/json/keychron-k6-shortcuts.json:26]: Why is this changed? This is not needed as it will be changed when this file is formatted.

Comment (sourabhv) [reply public/json/keychron-k6-shortcuts.json:25]: My formatter changed this, its fine if it will be reverted
